go.mod

```diff
@@ -6,4 +6,7 @@ require (
 	github.com/mattn/go-isatty v0.0.7 // indirect
 	github.com/sergi/go-diff v1.0.0
 	github.com/stretchr/testify v1.3.0 // indirect
+	golang.org/x/sys v0.0.0-20190531175056-4c3a928424d2 // indirect
 )
+
+go 1.13
```

Comment:
Forgive my ignorance, but what does this line mean?

I have trouble finding the relevant documentation.

Reply:
My bad, Go added it and I didn't notice.
I just pushed a commit to remove it. Just needed the updated version of the lib/